### ISSUE TITLE
Is it possible to use Horizon Chart as a row? [FeatureRequest]

### ISSUE THREAD

**Issue by OlegJakushkin** [wavedrom/wavedrom#288]
[Horizon Chart (with source code)](https://observablehq.com/@d3/realtime-horizon-chart) seems like a perfect addition to wavedrom capable to represent math functions and arbitrary data values. I wonder is it possible and how one can add a custom row renderer to wavedrom?

So what I would like to see
![image](https://user-images.githubusercontent.com/2915361/77805352-b86e6480-7092-11ea-843b-7fb7cce16e05.png)

And in terms of JSON, it would be something like
```json
{ signal: [
  { name: "func", wave: 'f.f(t){/*js code returning a float value*/ return Math.random() * t }' },
  { name: "vals", wave: 'v.x.345x', data: ["90", "300", "120"]  },
]}
```
But I focus primarily on heaving at least JS accessible renderer...
So is such thing possible and how one would do it?

**Comment by drom** [comment]
Well, at this point, WaveJson is just a data structure. And different tools, written in different languages JS, Python, Ruby, etc. can consume or produce it. Adding JS callback would definitely be a hostile move. Also, I am trying to imagine how "Horizon Chart" would be useful to Chip-Designer audience? There is a long-standing request on having "Analog" waves #189  #68 that might be related?

**Comment by OlegJakushkin** [comment]
Yes, it is in general alike what I want, rendering approaches do not matter.
